spec/policies/company_policy_spec.rb

```diff
@@ -24,4 +24,88 @@
     it { is_expected.to authorize(:destroy) }
     it { is_expected.to authorize(:update) }
   end
+
+  context 'for manager user' do
+    let(:user) { create(:user) }
+    let(:new_company) { create(:company) }
+
+    let!(:company_user) do
+      create(:company_user, :manager, company: new_company, user: user)
+    end
+
+    it { is_expected.not_to authorize(:create) }
+    it { is_expected.not_to authorize(:index) }
```

Comment:
The user should be able to view their companies. Let's say that the user is the owner of two different companies. In this case, we need to have an endpoint filtering those companies.